about/release_policy.rst

```diff
@@ -132,6 +132,43 @@ provided for testing purposes only.
     See :ref:`doc_upgrading_to_godot_4` for instructions on migrating a project
     from Godot 3.x to 4.x.
 
+Which version should I use for a new project?
+---------------------------------------------
+
+We recommend using Godot 4.x for new projects, as the Godot 4.x series will be
+supported long after 3.x stops receiving support in the future. One caveat is
+that a lot of third-party documentation hasn't been updated for Godot 4.x yet.
+If you have to follow a tutorial designed for Godot 3.x, we recommend keeping
+:ref:`doc_upgrading_to_godot_4` open in a separate tab to check which methods
+have been renamed (if you get a script error while trying to use a specific node
+or method that was renamed in Godot 4.x).
+
+If your project requires a feature that is missing in 4.x (such as GLES2/WebGL
+1.0), you should use Godot 3.x for a new project instead.
+
+Should I upgrade my project to use new engine versions?
+-------------------------------------------------------
+
+The general recommendation is to upgrade your project to follow new *patch*
```

Comment:
I would suggest that we start with a word of caution, that upgrades are inherently risky and it's better to evaluate the changes yourself. Then follow up with a notion that we do our best to keep minor and especially patch releases compatible. Then the rest of what you wrote.